docs/concepts/select.md

```diff
@@ -331,3 +294,85 @@ export class CityService {
 ```
 
 now we can use this `zooThemeParks` selector anywhere in our application.
+
+## Special Considerations
+
+### Angular Libraries: Use of lambdas in static functions
+
+_If you are building an Angular lib directly so that it can be deployed to npm. `strictMetadataEmit` will most likely be set to true in the Angular compiler options and, as a result, Angular's `MetadataCollector` from the `@angular/compiler-cli` package will report the following issue with using lambdas in static methods:_
```

Comment:
What about linking to https://angular.io/guide/aot-compiler#strictmetadataemit?

Reply:
Good point. Done 👍